source/hic_hal/freescale/k20dx/usb_config.c

```diff
@@ -345,23 +371,42 @@
 //     </e>
 #define USBD_CLS_ENABLE             0
 
+//     WebUSB support
+#ifdef WEBUSB_INTERFACE
+#define USBD_WEBUSB_ENABLE          1
+#else
+#define USBD_WEBUSB_ENABLE          0
+#endif
+#define USBD_WEBUSB_VENDOR_CODE     0x21
+#define USBD_WEBUSB_BASE_LANDING_URL "armmbed.github.io/dapjs-web-demo/?vid="
```

Comment:
This landing page needs to be replaced with os.mbed.com/webusb when it will be live.